server/internal/server/org.go

```diff
@@ -109,17 +109,36 @@ func (s *S) ListOrganizations(ctx context.Context, req *v1.ListOrganizationsRequ
 	}
 
 	// Only show orgs that the user is a owner/reader of.
-	var filtered []*store.Organization
+	var visibleOrgs []*store.Organization
 	for _, org := range orgs {
 		if s.organizationRole(org.OrganizationID, userInfo.UserID) != v1.OrganizationRole_ORGANIZATION_ROLE_UNSPECIFIED {
-			filtered = append(filtered, org)
+			visibleOrgs = append(visibleOrgs, org)
 		}
 	}
 
 	var orgProtos []*v1.Organization
-	for _, org := range filtered {
+	for _, org := range visibleOrgs {
 		orgProtos = append(orgProtos, org.ToProto())
 	}
+
+	if req.IncludeSummaries {
+		for _, org := range orgProtos {
+			ps, err := s.store.ListProjectsByTenantIDAndOrganizationID(userInfo.TenantID, org.Id)
```

Comment:
Actually not all the projects are visible to end users.. We would need to follow that logic we have in https://github.com/llmariner/user-manager/blob/main/server/internal/server/project.go#L143